scrapers_next/wv/committees.py

```diff
@@ -0,0 +1,151 @@
+        self.input.add_source(self.source.url, f"{name} url")
+        self.input.add_member(sub(r"\(.*\)", "", name), self.position)
+        return self.input
+
+    def get_source_from_input(self):
+        pass
+
+
+class CommitteeDetail(HtmlListPage):
+    """
+    Scrape a list of committee members.
+
+    :param selector: CSS matcher for each legislator url, according to chamber.
+    :param url_signifier: Part of each matched element's href that should exist
+    in a valid member profile url.
+    """
+
+    selector: CSS
+    url_signifier: str
+
+    def __init__(
+        self,
+        input_val: ScrapeCommittee,
+        source: Source,
+        selector: CSS,
+        url_signifier: str,
+    ):
+        super(CommitteeDetail, self).__init__(input_val, source=source)
+        self.selector = selector
+        self.url_signifier = url_signifier
+
+    def process_item(self, item):
+        if self.url_signifier in item.get("href"):
+            next_elem = item.getnext().text
+            position = next_elem if isinstance(next_elem, str) else "member"
+            return CommitteeMember(
+                self.input,
+                source=URL(item.get("href")),
+                position=position,
+                fallback_name=item.get("href").split("member=", 1)[1],
+            )
+        else:
+            raise SkipItem("not a committee member url")
+
+    def get_source_from_input(self):
+        pass
+
+
+class CommitteeList(HtmlListPage):
+    """
+    Scrape a list of committees.
+
+    :param selector: CSS matcher for each committee url, according to chamber.
+    :param member_selector: CSS matcher for each member within a committee -
+    passed directly to CommitteeDetail.
+    :param cmte_url_signifier: Part of each matched element's href that should exist
+    in a valid committee directory url.
+    :param member_url_signifier: Part of each matched element's href that should exist
+    in a valid member profile url.
+    :param chamber: Chamber of this group of committees. e.g. "upper" "lower" "legislature"
+    """
+
+    selector: CSS
+    member_selector: CSS
+    cmte_url_signifier: str
+    member_url_signifier: str
+    chamber: str
+
+    def process_item(self, item):
+        if self.cmte_url_signifier in item.get("href"):
+            cmte = ScrapeCommittee(
+                name=item.text,
+                chamber=self.chamber,
+                # WV subcommittees do not explicitly list parent committees
+                classification="committee",
+            )
+            cmte.add_source(item.get("href"), f"{item.text} committee url")
+            return CommitteeDetail(
+                input_val=cmte,
+                source=URL(item.get("href")),
+                selector=self.member_selector,
+                url_signifier=self.member_url_signifier,
+            )
+        else:
+            raise SkipItem("not a committee url")
+
+    def get_source_from_input(self):
+        pass
```

Comment:
You can remove this

Reply:
Addressed in [c75e495](https://github.com/openstates/openstates-scrapers/pull/4398/commits/c75e495355fb42fb500885f7e746621da1f2226c)